vaadin-select-flow-demo/src/main/java/com/vaadin/flow/component/select/SelectView.java

```diff
@@ -20,24 +26,69 @@
 import com.vaadin.flow.demo.DemoView;
 import com.vaadin.flow.router.Route;
 
-import java.util.List;
-import java.util.Objects;
-
 @Route("vaadin-select")
 public class SelectView extends DemoView {
 
+
+    // begin-source-example
+    // source-example-heading: Select example model
+    public static class Country {
+        private String name;
+        private String continent;
+        private String capital;
+
+        public Country(String name, String continent, String capital) {
+            this.name = name;
+            this.continent = continent;
+            this.capital = capital;
+        }
+
+        public String getName() {
+            return name;
+        }
+
+        public String getContinent() {
+            return continent;
+        }
+
+        public String getCapital() {
+            return capital;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) {
+                return true;
+            }
+            if (o == null || getClass() != o.getClass()) {
+                return false;
+            }
+            Country country = (Country) o;
+            return name.equals(country.name);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(name);
+        }
+    }
+
+    // end-source-example
+
+
     @Override
     protected void initView() {
         basicDemo();// Basic usage
         disabledAndReadonly();
         entityList();
         valueChanged();
         disabledItem();
-        configurationForReqiredDemo();// Validation
+        simpleDataNavigationDemo();
```

Comment:
Should the existing demos be updated to use data view approach too ? Would make sense IMO for the DX test or we might have a bad time there

Reply:
done